### PR TITLE
Fix docker system prune failing with Swarm mode disabled

### PR DESCRIPTION
**Pull request by thaJeztah**
Fixes https://github.com/docker/docker/issues/32147

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @aboch @aaronlehmann for naming of the interfaces I added 😅

**Review by vdemeester — APPROVED** [review]
LGTM 🐯

File: daemon/cluster.go [hunk 1]
```diff
@@ -6,6 +6,18 @@ import (
 
 // Cluster is the interface for github.com/docker/docker/daemon/cluster.(*Cluster).
 type Cluster interface {
+	ClusterStatus
+	NetworkManager
+}
+
+// ClusterStatus interface provides information about the Swarm status of the Cluster
```
Comment (ehazlett) [inline daemon/cluster.go]: nit: `go vet` won't like this being different than the func name.

File: daemon/cluster.go [hunk 2]
```diff
@@ -6,6 +6,18 @@ import (
...
+type ClusterStatus interface {
+	IsAgent() bool
+	IsManager() bool
+}
+
+// ClusterNetworkManager provides methods to manage networks
```
Comment (ehazlett) [inline daemon/cluster.go]: nit: `go vet` won't like this being different than the func name.

Comment (thaJeztah) [inline daemon/cluster.go]: oh! I was having an internal bike-shed over the name, decided that more "generic" was better. Let me update, thanks!

Comment (thaJeztah) [inline daemon/cluster.go]: fixed!

**Review by AkihiroSuda — APPROVED** [review]
LGTM

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Pull request merged**